aiozipkin/aiohttp_helpers.py

```diff
@@ -89,3 +92,52 @@ def get_tracer(app, tracer_key=APP_AIOZIPKIN_KEY):
 
 def request_span(request, request_key=REQUEST_AIOZIPKIN_KEY):
     return request[request_key]
+
+
+def _has_span(trace_config_ctx):
+    trace_request_ctx = trace_config_ctx.trace_request_ctx
+    has_span = (isinstance(trace_request_ctx, dict) and
+                'span_context' in trace_request_ctx)
+    return has_span
+
+
+def make_trace_config(tracer):
```

Comment:
It's up to you but the function looks like a class written with JavaScript techniques.

Reply:
agreed will fix

Reply:
fixed